usecase/parser/v0_43_0/ibcmsg/ibcmsg.go

```diff
@@ -0,0 +1,150 @@
+package ibcmsg
+
+import (
+	"fmt"
+	"time"
+
+	base64_internal "github.com/crypto-com/chain-indexing/internal/base64"
+	"github.com/crypto-com/chain-indexing/internal/json"
+	"github.com/crypto-com/chain-indexing/internal/primptr"
+	"github.com/crypto-com/chain-indexing/usecase/parser/ibcmsg"
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/crypto-com/chain-indexing/entity/command"
+	"github.com/crypto-com/chain-indexing/internal/typeconv"
+	command_usecase "github.com/crypto-com/chain-indexing/usecase/command"
+	ibc_model "github.com/crypto-com/chain-indexing/usecase/model/ibc"
+	"github.com/crypto-com/chain-indexing/usecase/parser/utils"
+)
+
```

Comment:
Maybe better to document what has been updated in this parser for future reference

Reply:
MsgRecvPacket success inverted value should be fixed in v0.42.7 instead of v0.43 though